docs/src/components/heading.tsx

```diff
@@ -1,12 +1,19 @@
+import slugify from '@sindresorhus/slugify';
 import { ReactNode } from 'react';
+import { LinkIcon } from '@heroicons/react/24/outline';
 
 type HeadingProps = {
   level: 1 | 2 | 3 | 4 | 5 | 6;
   children: ReactNode;
+  anchor?: boolean;
 };
 type ValidHeading = 'h1' | 'h2' | 'h3' | 'h4' | 'h5' | 'h6';
 
-export default function Heading({ level, children }: HeadingProps) {
+export default function Heading({
+  level,
+  children,
+  anchor = false,
```

Comment:
I don't think you need the `= false`, do you?

Reply:
- [ ] anchor = false